### ISSUE TITLE
Support Lambda images

### ISSUE THREAD

**Issue by matthewhanson** [cirrus-geo/cirrus-geo#139]
- Update serverless to at least 2.72
- update path in API proxy
- remove magic addition of module and handler during build (put in template)

**Comment by jkeifer** [comment]
So if we want to support serverless >=3.0, we have to remove the `serverlerss-pseudo-parameters` plugin. The replacement on the serverless side only supports two common parameters, region and account id. For every other use, we have to convert to using `Fn:Sub`.

Interestingly, we can easily process strings using the same regex as the aforementioned plugin, looking for things within a string that match the pattern `#{AWS::[^}]+}` and replacing the `#` with a `$`. If not already inside a `Fn:Sub` then we can add that.

Now, the concern becomes the fact that serverless also uses `${...}` for its variable interpolation, but good news: as of 2.3.0, the parser is smart enough to leave the AWS parameters alone (see https://github.com/serverless/serverless/releases/tag/v2.3.0 and https://github.com/serverless/serverless/pull/8279).

In other words, we can implement our own mini version of the pseudo parameters plugin. We could also consider logging a deprecation warning with details how to get off the `#{...}` syntax. But for now we'll just make things work without the plugin and see how that goes.

**Comment by jkeifer** [comment]
The removal of the module/handler magic will now result in a `ValueError` if the module is not defined explicitly in a lambda task `definition.yml`. The error looks something like this:

```
ValueError: Missing module parameter in lambda definiton.yml. The handler parameter, which sets the module, is no longer defaulted. You likely need to set it to be like:

lambda:
  handler: lambda_function.lambda_handler

Offending <lambda_type>: <lambda_name>

See https://github.com/cirrus-geo/cirrus-geo/issues/139 for additional context.
```
